internal/configload/configload.go

```diff
@@ -36,20 +36,23 @@ type config struct {
 	Dir     string        `yaml:"dir"`
 	Args    []string      `yaml:"args"`
 	Results struct {
-		Common   *testConfig `yaml:"common"`
-		FerretDB *testConfig `yaml:"ferretdb"`
-		MongoDB  *testConfig `yaml:"mongodb"`
+		Includes map[string][]string `yaml:"includes"`
+		Common   *testConfig         `yaml:"common"`
+		FerretDB *testConfig         `yaml:"ferretdb"`
```

Comment:
```suggestion
		PostgreSQL *testConfig         `yaml:"ferretdb"`
```

Preserve the YAML tag for compatibility for now. We will update it later everywhere.